packages/config/src/lib/config.test.js

```diff
@@ -76,18 +82,7 @@ describe('rootFolder', () => {
 describe('module rules', () => {
   test('length', () => {
     const { module } = configBuilder({ appEntry: 'testEntry', appName: 'someName' });
-    expect(module.rules.length).toBe(6);
-  });
-
-  test('first to be chrome-render-loader', () => {
-    const { module } = configBuilder({ appEntry: 'testEntry', appName: 'someName' });
-    expect(new RegExp(module.rules[0].rules).test('testEntry')).toBe(true);
-    expect(module.rules[0].options.skipChrome2).toBe(false);
-  });
-
-  test('first to be chrome-render-loader', () => {
-    const { module } = configBuilder({ appEntry: 'testEntry', appName: 'someName', skipChrome2: true });
-    expect(module.rules[0].options.skipChrome2).toBe(true);
+    expect(module.rules.length).toBe(4);
```

Comment:
Do we really need to check the _exact_ length here? This just seems fragile to me if we add other rules in the future. Maybe we should confirm the rules.length > 0